test/hocr-wordfreq/hocr-wordfreq.tsht

```diff
@@ -4,10 +4,10 @@ TESTDATA="../testdata"
 plan 4
 
 normal=$(hocr-wordfreq $TESTDATA/sample.html | head -n 1)
-casesensitive=$(hocr-wordfreq -i $TESTDATA/sample.html | head -n 1)
+caseinsensitive=$(hocr-wordfreq -i $TESTDATA/sample.html | head -n 1)
```

Comment:
Which use case do you have in mind? Word lists for OCR or spell checkers need the correct case.